doc/configuration.adoc

```diff
@@ -220,6 +220,8 @@ Oxalis uses a shared connection pool of keep-alive connections to speed up trans
 ----
 oxalis.http.pool.max_route = 2
 oxalis.http.pool.total = 20
+oxalis.http.pool.validate_after_inactivity = 1000
```

Comment:
I suggest descreasing this value from 1000s to at 60s or even less. Azure LBs starts timing out TCP connections at 4 minutes / 240 seconds. Other cloud vendors might have even more aggressive defaults.

See https://docs.microsoft.com/en-us/azure/load-balancer/load-balancer-tcp-idle-timeout?tabs=tcp-reset-idle-portal